app/controllers/articles_controller.rb

```diff
@@ -45,7 +45,11 @@ def redirect_hatena_blog_entries_url(params)
 
     redirect_to("/#{article.url}", status: 301) and return if article
 
-    render(file: "#{Rails.root}/public/404.html", status: 404, layout: false,
-      content_type: 'text/html')
+    render(
+      file: "#{Rails.root}/public/404.html",
+      status: 404,
+      layout: false,
+      :content_type => 'text/html'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.